.github/workflows/update-foundry-bin-monthly.yml

```
@@ -0,0 +1,24 @@
+name: Update foundry-bin monthly
+
+on:
+  workflow_dispatch:
+  schedule:
+  - cron: '6 9 5 * *' # Run daily
```

```suggestion
  - cron: '6 9 2 * *' # Run monthly
```

Mainly update comment, but also any particular reason you went with 5th day rather than earlier?

---

no really, largely a random choice biased towards more "buffer" time for whatever error can occur in between.

---

updated